### ISSUE TITLE
No syntax highlighting and auto completion for html and beans

### ISSUE THREAD

**Issue by Olafxso** [ddekany/jbosstools-freemarker#5]
It looks like the editor only has got syntax highlighting and auto completion for Freemarker tags and function ```<# >```. The syntax highlighting and auto completion for html tags is not working any more. Is it possible to get them back? When you type in de default html editor for ```<di``` you can press enter and you get ```<div></div>```. 
It would also very helpfull if you can ctrl+click to a variable like this.  ```${latestProduct.url} ```
Is this a know issue?

**Comment by ddekany** [comment]
Yeah, there's no HTML highlighting in it. I think the architecture of this plugin won't allow reusing the existing HTML editor functionality in Eclipse (like in WTP).

Clicking... I don't know if it ever had it. Maybe not. I was really just forking it to do some very basic maintenance (compatibility with new Eclipse, upgrade FreeMarker, etc.), so it remains as functional as it was at least.

Contributions of such features are highly welcome of course.
